packages/react-core/src/demos/DataList/examples/DataListActionable.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {
+  Button,
+  Content,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Dropdown,
+  DropdownList,
+  DropdownItem,
+  MenuToggle,
+  MenuToggleElement,
+  PageSection,
+  PageSectionVariants,
+  Title
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+export const DataListActionable: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isDeleted, setIsDeleted] = React.useState(false);
+
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <DashboardWrapper mainContainerId="main-content-datalist-view-actions" breadcrumb={null}>
+      <PageSection variant={PageSectionVariants.secondary}>
+        <Content>
+          <Title headingLevel="h1">Projects</Title>
+          <Content component="p">This is a demo that showcases PatternFly Data List</Content>
+        </Content>
+      </PageSection>
+      <DataList aria-label="single action data list example ">
```

Comment:
we should wrap this data list in a `<PageSection />` - components aren't designed to be a direct child of the page main container.